lib/widgets/content.dart

```diff
@@ -1586,8 +1586,9 @@ InlineSpan _errorUnimplemented(UnimplementedNode node, {required BuildContext co
       TextSpan(text: "»)", style: errorStyle),
     ]);
   } else {
+    final zulipLocalizations = ZulipLocalizations.of(context);
     return TextSpan(
-      text: "(unimplemented: DOM node type ${htmlNode.nodeType})",
+      text: zulipLocalizations.errorUnimplementedPlaceholder(htmlNode.nodeType),
```

Comment:
I think this one is covered by the TODO comments above, so we can leave it out.

(I'm not sure I've ever seen this case happen, so there's no urgency to ask our translators to translate it before we have a solution for its neighbors. The `is dom.Element` case above is by far the most common.)